Sources/Data/models/PlaylistInfo.swift

```diff
@@ -74,7 +74,7 @@ public struct PlaylistInfo: Codable, Identifiable, Hashable, Equatable {
   public init(from decoder: Decoder) throws {
     let container = try decoder.container(keyedBy: CodingKeys.self)
     self.name = try container.decode(String.self, forKey: .name)
-    let src = try container.decode(String.self, forKey: .src)
+    let src = try container.decode(String?.self, forKey: .src) ?? ""
```

Comment:
Probably better to use `decodeIfPresent` over passing in an optional type

Reply:
Fixed.